dvc/output/base.py

```diff
@@ -328,6 +332,8 @@ def dumpd(self):
         if self.checkpoint:
             ret[self.PARAM_CHECKPOINT] = self.checkpoint
 
+        ret[self.PARAM_IS_USER_EXECUTABLE] = self.is_user_executable
+
```

Comment:
same as in other fields above
```suggestion
        if self.is_user_executable:
            ret[self.PARAM_IS_USER_EXECUTABLE] = self.is_user_executable
        
```